File: dbflow/src/androidTest/java/com/raizlabs/android/dbflow/test/structure/externalforeignkey/ForeignParentModel.java
package com.raizlabs.android.dbflow.test.structure.externalforeignkey;

import com.raizlabs.android.dbflow.annotation.ModelContainer;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.test.structure.TestModel1;

/**
 * Description:
 */
@ModelContainer
@Table(database=TestDatabase2.class)
public class ForeignParentModel extends TestModel1 {
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/structure/foreignkey/ForeignParentModel.kt
package com.raizlabs.android.dbflow.structure.foreignkey

import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.TestDatabase
import com.raizlabs.android.dbflow.structure.TestModel1

/**
 * Description:
 */
@Table(database = TestDatabase::class)
open class ForeignParentModel : TestModel1()
